samples/nrf9160/nrf_cloud_mqtt_multi_service/README.rst

```diff
@@ -8,7 +8,7 @@ nRF9160: nRF Cloud MQTT multi-service
    :depth: 2
 
 This sample is a minimal, error tolerant, integrated demonstration of the :ref:`lib_nrf_cloud`, :ref:`lib_location`, and :ref:`lib_at_host` libraries.
-It demonstrates how you can use these libraries together to support Firmware-Over-The-Air (FOTA), location services, modem AT commands over UART, and periodic sensor samples in your `nRF Cloud`_-enabled application.
+It demonstrates how you can use these libraries together to support Firmware-Over-The-Air (FOTA), location services, modem AT commands over UART and MQTT, application-specific device messages, and periodic sensor samples in your `nRF Cloud`_-enabled application.
```

Comment:
```suggestion
It demonstrates how you can use these libraries together to support the following in your `nRF Cloud`_-enabled application:

* Firmware-Over-The-Air (FOTA)
* Location Services
* Modem AT commands over UART and MQTT
* Application-specific device messages
* Periodic sensor samples

```

Reply:
Since I have a more detailed list below, maybe it would be better to switch to a less detailed overview here -- perhaps:

> It demonstrates how you can use these libraries together to support Firmware-Over-The-Air (FOTA), location services, periodic sensor samples, and more in your `nRF Cloud`_-enabled application.

Do you think that would work?